src/ontology/ro-edit.owl

```diff
@@ -6422,8 +6430,8 @@ SubObjectPropertyOf(obo:RO_0016002 obo:RO_0016001)
 
 AnnotationAssertion(obo:IAO_0000115 obo:RO_0016004 "X has exposure medium Y if X is an exposure event (process), Y is a material entity, and the stimulus for X is transmitted or carried in Y.")
 AnnotationAssertion(obo:IAO_0000119 obo:RO_0016004 "ExO:0000083")
-AnnotationAssertion(dce:date obo:RO_0016004 "2021-12-14T20:41:45Z"^^xsd:dateTime)
-AnnotationAssertion(oboInOwl:created_by obo:RO_0016004 <https://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(dc:creator obo:RO_0016004 <https://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(dc:date obo:RO_0016004 "2021-12-14T20:41:45Z")
```

Comment:
Why was the xsd:dateTime removed?

Reply:
This might be connected: Something to be on the watch for - Protege 5.6.1 seems to have a bug in it such that if you edit an annotation, the form it pops up shows "xsd:string" as default datatype even if existing content has another datatype.  So editing an existing time entry - even just looking at one and saving it, causes the datatype to be overwritten (/cleared out). See https://github.com/protegeproject/protege/issues/1152